.github/workflows/all-green.yml

```diff
@@ -0,0 +1,26 @@
+name: Check Pull Request CI Status
+
+on: # yamllint disable-line rule:truthy
+  pull_request:
+    types:
+      - opened
+      - synchronize
+      - reopened
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+permissions:
+  checks: read
+  statuses: read
+
+jobs:
+  all-jobs-are-green:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Run Ensure CI Success
+        uses: DataDog/ensure-ci-success@f40e6ffd8e60280d478b9b92209aaa30d3d56895
+        with:
+          initial-delay-seconds: "600"
```

Comment:
1. The value should be string. 
2. A better value for this repository is 15 minute.
3. Rerunning the job should skip the delay.